catalog-info.yaml

```diff
@@ -5,15 +5,36 @@ metadata:
   description: API for creating and deploying applications on the VA Platform.
   annotations:
     github.com/project-slug: department-of-veterans-affairs/platform-console-api
+    backstage.io/techdocs-ref: url:https://github.com/department-of-veterans-affairs/platform-console-api
   tags:
     - ruby
     - rails
     - platform-console-api
+  links:
+    - url: http://platform-console-api.vfs.va.gov
+      title: Platform Console API - Utility
```

Comment:
Did you add these items or is this a merge conflict?

Reply:
This was removed.